alarm_trap_sender.hpp

```diff
@@ -170,7 +184,7 @@ class AlarmTrapSender
   void sync_alarms(bool do_clear);
 
   static AlarmTrapSender& get_instance() {return _instance;}
-
+  alarmActiveTable_SNMPDateTime* alarm_time_issued(void);
```

Comment:
As it's only the active alarm table that needs the time in this precise format, should this be part of the active alarm table rather than part of the trap sender?